### ISSUE TITLE
Request to Tunisia for FoxOs image

### ISSUE THREAD

**Issue by akanshgulati** [Mozpacers/Mozpacers.github.io#68]
The css file in client directory requesting tunisia for an image

`#panel-fxos {
  background: #0095dd url("http://mozilla-tunisia.org/static/home/img/firefoxos-fox.png") -100px -80px no-repeat;
}`

 Upload it to server as its taking a lot of time to get a response.

**Comment by handasolo** [comment]
I don't think we need rights for this. It's a FirefoxOS image and can be used by us. Someone can simply save this image and shift to our local repository. Easy good first bug to fix for people.
